app/Repositories/ArtworkRepository.php

```diff
@@ -8,11 +8,15 @@
 use A17\Twill\Repositories\ModuleRepository;
 use App\Libraries\Api\Builders\ApiQueryBuilder;
 use App\Models\Artwork;
+use App\Repositories\Traits\HandleDynamicArtworkFields;
 use Illuminate\Support\Str;
 
 class ArtworkRepository extends ModuleRepository
 {
-    use HandleMedias, HandleRevisions, HandleTranslations;
+    use HandleDynamicArtworkFields;
```

Comment:
Saving artwork was causing a `Unknown column 'thumbnail' in 'field list'` errors.
These trait methods are called by the base `ModuleRepository` to clean up the fields before saving or updating.